docs/MockFunctionAPI.md

```diff
@@ -116,7 +117,8 @@ don't access stale data.
 
 ### `mockFn.mockRestore()`
 
-Removes the mock and restores the initial implementation.
+Does everything that [`mockFn.mockReset()`](#mockfnmockreset) does, and also
+restores the original (non-mocked) implementation.
```

Comment:
Should this sayrestores the original (non-mocked) implementation _for spies_`

Reply:
Yeah, that probably makes more sense 👍